suites/squid/nvmeof/tier-1_nvmeof_gwgroup_sanity.yaml

```diff
@@ -0,0 +1,229 @@
+      abort-on-fail: false
+      config:
+        verify_cluster_health: true
+        steps:
+          - config:
+              command: shell
+              args:
+                - ceph osd pool create rbd
+          - config:
+              command: shell
+              args:
+                - rbd pool init rbd
+          - config:
+              command: apply
+              service: nvmeof
+              args:
+                placement:
+                  nodes:
+                    - node6
+                    - node7
+              pos_args:
+                - rbd
+                - group1
+          - config:
+              command: apply
+              service: nvmeof
+              args:
+                placement:
+                  nodes:
+                    - node8
+                    - node9
+              pos_args:
+                - rbd
+                - group2
+          - config:
+              command: apply
+              service: nvmeof
+              args:
+                placement:
+                  nodes:
+                    - node10
+                    - node11
+              pos_args:
+                - rbd
+                - group3
+          - config:
+              command: apply
+              service: nvmeof
+              args:
+                placement:
+                  nodes:
+                    - node12
+                    - node13
+              pos_args:
+                - rbd
+                - group4
+      desc: deploy NVMeoF services on 4 GW groups having 4 GW nodes
+      destroy-cluster: false
+      do-not-skip-tc: true
+      module: test_cephadm.py
+      name: deploy NVMeoF services on 4 GW groups having 8 GW nodes
+
+  - test:
+      abort-on-fail: true
+      config:
+        rbd_pool: rbd
+        do_not_create_image: true
+        rep-pool-only: true
+        rep_pool_config:
+          pool: rbd
+        cleanup:
+          - pool
+          - gateway
+          - initiators
+        gw_groups: # Configure gatewayGroups
+          - gw_group: group1
+            gw_nodes:
+              - node6
+              - node7
+            subsystems: # Configure subsystems with all sub-entities
+              - nqn: nqn.2016-06.io.spdk:cnode10
+                listener_port: 4420
+                listeners: [node6, node7]
+                allow_host: "*"
+                bdevs:
+                  - count: 2
+                    size: 2G
+              - nqn: nqn.2016-06.io.spdk:cnode2
+                listener_port: 4420
+                listeners: [node6, node7]
+                allow_host: "*"
+                bdevs:
+                  - count: 2
+                    size: 2G
+          - gw_group: group2
+            gw_nodes:
+              - node8
+              - node9
+            subsystems:
+              - nqn: nqn.2016-06.io.spdk:cnode3
+                listener_port: 4420
+                listeners: [node8, node9]
+                allow_host: "*"
+                bdevs:
```

Comment:
This would be hard to deploy namespace with particulat load-balancing-group-id.

Reply:
Added load-balancing-group-id under bdevs section for each subsystem.